tools/Dockerfile.golang

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.6.2
+
+RUN apt-get update \
+    && apt-get install -y libldap2-dev \
+    && rm -r /var/lib/apt/lists/*
+
+#golang.org is blocked in China
+COPY ./src/vendor/golang.org /go/src/golang.org 
+
+RUN go get -d github.com/docker/distribution \
```

Comment:
I don't think you need such thing the dependencies have been moved to vendor/ folder for quite a while so no go get should be needed....